lib/router/handlers/serve_static.ex

```diff
@@ -12,6 +14,13 @@ defmodule HTTPServer.Router.Handlers.ServeStatic do
     end
   end
 
+  def add_static_dir(routes, dirpath, path \\ "") do
+    dirpath
+    |> get_filepaths()
+    |> build_routes(dirpath, path)
+    |> Map.merge(routes)
+  end
+
```

Comment:
with the path defaulting to `""` is this the same as root ("\").  With this function, could you also add more than one "static directory" to a particular set of routes?  ... all good things to unit test.

Reply:
Great idea! I was struggling to come up with unit tests to create that didn't retest functionality the integration test was already going to address, so this is helpful.